instrumentation/spring/spring-webflux/spring-webflux-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/v5_0/server/HandlerAdapterInstrumentation.java

```diff
@@ -65,8 +67,17 @@ public static void methodEnter(
 
       Context parentContext = Context.current();
 
-      HttpRouteHolder.updateHttpRoute(
```

Comment:
This isn't really where the problem is. Server span name is computed from the http route. You get the wrong server span name because the route is wrong. You really should be prepending the context path to the route similarly to how it is done in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/eac75a1d0b63fb2e1d276b8ece6523592924d087/instrumentation/spring/spring-webmvc/spring-webmvc-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webmvc/v3_1/SpringWebMvcServerSpanNaming.java#L20
You could also modify one of the tests to use a non-empty context path to verify your fix works as intended.